.github/workflows/release.yml

```diff
@@ -223,33 +223,29 @@ jobs:
                   pnpm run lint
 
     ####### Building and Testing binaries #######
-
     build:
         runs-on: self-hosted
         needs: ["set-tags"]
         env:
             TMP_TARGET: "/tmp/target"
             CARGO_TARGET_DIR: "target"
+            RUSTC_WRAPPER: "sccache"
+            CARGO_INCREMENTAL: "0"
+            SCCACHE_CACHE_SIZE: "100GB"
+            SCCACHE_GHA_ENABLED: "true"
```

Comment:
We didn't have `SCCACHE_GHA_ENABLED` before, what does that do?

Reply:
GHA native sccache, haven't fully explored options with this:

https://github.com/mozilla/sccache/blob/main/docs/GHA.md